torchvision/transforms/_pil_constants.py

```diff
@@ -0,0 +1,22 @@
+
+import PIL
+from PIL import Image
+
+if tuple(int(part) for part in PIL.__version__.split(".")) >= (9, 1, 0):
+    FLIP_LEFT_RIGHT = Image.Transpose.FLIP_LEFT_RIGHT
+    FLIP_TOP_BOTTOM = Image.Transpose.FLIP_TOP_BOTTOM
+    BILINEAR = Image.Resampling.BILINEAR
+    NEAREST = Image.Resampling.NEAREST
+    AFFINE = Image.Transform.AFFINE
+    BICUBIC = Image.Resampling.BICUBIC
+    PERSPECTIVE = Image.Transform.PERSPECTIVE
+    LINEAR = Image.Resampling.BILINEAR
```

Comment:
Could you order them by module and then alphanumeric? 

```suggestion
    BICUBIC = Image.Resampling.BICUBIC
    BILINEAR = Image.Resampling.BILINEAR
    NEAREST = Image.Resampling.NEAREST
    LINEAR = Image.Resampling.BILINEAR
    
    AFFINE = Image.Transform.AFFINE
    FLIP_LEFT_RIGHT = Image.Transpose.FLIP_LEFT_RIGHT
    FLIP_TOP_BOTTOM = Image.Transpose.FLIP_TOP_BOTTOM
    PERSPECTIVE = Image.Transform.PERSPECTIVE
```

Same for the other branch please.